src/main/java/chatbot/Ui.java

```diff
@@ -0,0 +1,58 @@
+package chatbot;
+
+import chatbot.task.Task;
+
+import java.util.Scanner;
+
+/**
+ * Class which deals with interactions with the user.
+ */
+public class Ui {
+    private Scanner scanner;
+
+    /**
+     * constructor for Ui class.
+     */
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Read command.
+     *
+     * @return String of the command
+     */
+    public String readCommand() {
+        return scanner.nextLine().trim();
+    }
+
+    /**
+     * print greeting message.
+     */
+    public void showGreeting() {
+        System.out.println("    ____________________________________________________________");
```

Comment:
Do you think it would be better if the System.out.println(<line>) was abstracted into a "linePrinter" function, since it is called a couple of times?